server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -257,7 +259,18 @@ public Invocations getInvocations() {
          cachesHealth.put(ch.getCacheName(), ch.getStatus());
       }
 
+      LocalTopologyManager localTopologyManager = null;
+      Boolean clusterRebalancingEnabled = null;
+      try {
+         localTopologyManager = SecurityActions.getGlobalComponentRegistry(cacheManager).getLocalTopologyManager();
+         clusterRebalancingEnabled = localTopologyManager.isRebalancingEnabled();
+      } catch (Exception e) {
+         // Unable to get the component. Just ignore.
+      }
+
       // We rely on the fact that getCacheNames doesn't block for embedded - remote it does unfortunately
+      LocalTopologyManager finalLocalTopologyManager = localTopologyManager;
```

Comment:
You wouldn't need new variables if you initialized `localTopologyManager` and `clusterRebalancingEnabled` inside the try + catch blocks.